assets/src/dashboard/components/popover-panel/index.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as CloseIcon } from '../../icons/close.svg';
+import { Z_INDEX } from '../../constants';
+
+export const Panel = styled.div`
+  align-items: flex-start;
+  background-color: ${({ theme }) => theme.colors.white};
+  border-radius: 8px;
+  box-shadow: 0px 4px 14px rgba(0, 0, 0, 0.25);
+  display: flex;
+  flex-direction: column;
+  margin: ${({ framelessButton }) => (framelessButton ? '0' : '20px 0')};
+  opacity: ${({ isOpen }) => (isOpen ? 1 : 0)};
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  pointer-events: ${({ isOpen }) => (isOpen ? 'auto' : 'none')};
+  transform: ${({ isOpen }) =>
+    isOpen ? 'translate3d(0, 0, 0)' : 'translate3d(0, -1rem, 0)'};
+  z-index: ${Z_INDEX.POPOVER_MENU};
```

Comment:
nit: even if this is the same z index i feel like it should be its own key just incase things change down the line to avoid tangling up variants of the popover?